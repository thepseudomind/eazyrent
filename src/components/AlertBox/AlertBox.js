import React from 'react';
import './AlertBox.css';

const AlertBox = ({message})=>{
    switch (message) {
        case 'Bank already added':
            return (
                <div className="alert-box">
                    <p className="alert-box__message">Bank already added</p>
                </div>
            );
        
        case 'Maximum of 3 banks allowed':
                return (
                    <div className="alert-box">
                        <p className="alert-box__message">Maximum of 3 banks allowed</p>
                    </div>
                );

        case 'Bank details changed':
                return (
                    <div className="alert-box">
                        <p className="alert-box__message">Bank details changed</p>
                    </div>
                );
    
        default:
            return <span></span>;
    }
}

export default AlertBox;