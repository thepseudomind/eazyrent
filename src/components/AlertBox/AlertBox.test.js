import { shallow } from 'enzyme';
import React from 'react';
import AlertBox from './AlertBox';

it('testing AlertBox', ()=>{
    const message = 'dummy';
    expect(shallow(<AlertBox message={message}/>)).toMatchSnapshot();
    expect(message).toBe('dummy');
})