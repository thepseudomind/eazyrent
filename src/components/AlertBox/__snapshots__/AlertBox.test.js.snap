// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`testing AlertBox 1`] = `ShallowWrapper {}`;
